src/Authentication/Provider/JwtAuth.php

```diff
@@ -62,8 +62,8 @@ public function __construct(EntityTypeManagerInterface $entity_type_manager, Jwt
    * {@inheritdoc}
    */
 	public function applies(Request $request) {
-    $auth = $request->headers->get('Authorization');
-		return preg_match('/^Bearer .+/', $auth);
+      $auth = $request->headers->get('Authorization');
+      return preg_match('/^Bearer .+/', $auth);
```

Comment:
Thank you for all the work you're doing! This is great. One nit: Can you fix the indentation in this file?